### DISCUSSION TITLE
Connect to milvus server using general http address?

### DISCUSSION THREAD

**Discussion by kakkoiiAllen** [milvus-io/milvus#18499] [Q&A and General discussion]
Hi:
I have a milvus server running on http://xxx/xxx/milvus-server, the exact address(ip:port) of the server is managed by the load balancer and the container orchestration tools. I tried to connect using the http address but it doesn't work. I don't want to hack for the ip address to add extra latency too. Is there a way to connect to the server using the http address? 
Thank you

**Comment by yhmo** [comment]
Some users deploy milvus as a local service, and wrap the service by web framework such as Flask/FastAPI. Customer use web browser to connect the web application by http address, the web server invoke milvus service API.
